src/@orb.yml

```diff
@@ -10,4 +10,4 @@ display:
 # if our orb requires other orbs, we can import them like this. Otherwise remove the "orbs" stanza.
 orbs:
   node: circleci/node@5
-  browser-tools: circleci/browser-tools@1.4.5
+  browser-tools: circleci/browser-tools@1.4.6
```

Comment:
Maybe it makes sense to have like this 
`circleci/browser-tools@1` 

or at least like this
`circleci/browser-tools@1.4`

So we will automatically receive all minor updates and don't need to change version manually every time
https://circleci.com/docs/orb-concepts/#semantic-versioning